src/config.js

```diff
@@ -0,0 +1,33 @@
+define(['lodash-amd/modern/objects/defaults',], function (defaults) {
+
+  var defaultOptions = {
+    allowBlockElements: true,
+    debug: false,
+    undo: {
+      manager: false,
+      enabled: true,
+      limit: 100,
+      interval: 250
+    },
+    defaultCommandPatches: [
+      'bold',
+      'indent',
+      'insertHTML',
+      'insertList',
+      'outdent',
+      'createLink'
+    ]
+  };
+
+
+  function checkOptions(userSuppliedOptions) {
+    var options = userSuppliedOptions || {};
+
+    return Object.freeze(defaults(options, defaultOptions));
```

Comment:
Well the tests still pass so I don't think we really understand the FF31/32 undo manager issue. But yes the plan is to depreciate support so at that point we won't need to try and re-write window.scribe.options (which we really don't want to happen).